### PR TITLE
Renamed and scoped npm module to @fusionauth/react-sdk

### PR DESCRIPTION
**Pull request by bhalsey**
## What is this PR and why do we need it?

Now scoping npm module under `@fusionauth` for better maintainability.

#### Pre-Merge Checklist (if applicable)

-   [x] Unit and Feature tests have been added/updated for logic changes, or there is a justifiable reason for not doing so.

### CODE REVIEW

File: CHANGES.adoc [hunk 1]
```diff
@@ -0,0 +1,9 @@
+fusionauth-react-sdk Changes
+
+Changes in 0.22.2
+
+ * *Breaking change* Module is now scoped under `@fusionauth`.  Users will need to update `package.json` and imports.
+
+Changes in 0.22.1
```
Comment (mooreds) [inline CHANGES.adoc:7]: Can we bump at least the minor version here, if not the major version? 

semver is something we try to implement.

Comment (robotdan) [reply CHANGES.adoc:7]: > Major version zero (0.y.z) is for initial development. Anything MAY change at any time. The public API SHOULD NOT be considered stable.

Technically this means we can change anything we want in a dot or patch version. Once we get to a stable version, we should then transition to version 1 and proceed with standard versioning scheme. 

All that said, I am fine using `0.22.1` or `0.23.0` here.

Comment (mooreds) [reply CHANGES.adoc:7]: Fair. Increment however you want, then @bhalsey .

Comment (bhalsey) [reply CHANGES.adoc:7]: Went with 0.23.0.  Thanks for clarifications

**Review by mooreds — APPROVED** [review]

**Pull request merged**